lambdas/services/document_manifest_service.py

```diff
@@ -36,11 +41,22 @@ def create_document_manifest_presigned_url(
         self, doc_type: SupportedDocumentTypes
     ) -> str:
         try:
+            filter_expression = filter_expression_for_available_docs()
             documents = (
                 self.document_service.fetch_available_document_references_by_type(
-                    nhs_number=self.nhs_number, doc_type=doc_type
+                    nhs_number=self.nhs_number,
+                    doc_type=doc_type,
+                    query_filter=filter_expression,
                 )
             )
+            file_in_progress_message = (
+                "The patients Lloyd George record is in the process of being uploaded"
+            )
+
+            for doc in documents:
```

Comment:
Clean coding perspective, try not to abbreviate. 'document' would be better than 'doc'.